src/pages/ReimbursementAccount/NonUSD/WhyLink.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type WhyLinkProps = {
+    /** Style for wrapping View */
+    containerStyles?: StyleProp<ViewStyle>;
+};
+
+function WhyLink({containerStyles}: WhyLinkProps) {
```

Comment:
This component was already added in @pasyukevich's PR here: https://github.com/Expensify/App/pull/52054/files

Reply:
Oh It was merged recently, I didn't notice it during this PR review process